_Boards/Atmel/Board_ProMicro/MFBoards.h

```diff
@@ -1,21 +1,20 @@
 #ifndef MFBoardMicro_h
 #define MFBoardMicro_h
 
-#define MF_SEGMENT_SUPPORT 0
-#define MF_LCD_SUPPORT 0
-#define MF_STEPPER_SUPPORT 0
-#define MF_SERVO_SUPPORT 0
+#define MF_SEGMENT_SUPPORT 1
+#define MF_LCD_SUPPORT 1
+#define MF_STEPPER_SUPPORT 1
+#define MF_SERVO_SUPPORT 1
 #define MF_ANALOG_SUPPORT 1
-#define MF_SHIFTER_SUPPORT 0
-#define MF_INPUT_SHIFTER_SUPPORT 1
```

Comment:
This was an error on my part, accidentally having a #define in the wrong branch.